docs/book/v1/how-it-works.md

```diff
@@ -80,7 +80,7 @@ be freed after a request. This allows application configuration and artifacts
 (such as middleware and handlers) to persist between requests and processes.
 
 Under Swoole 4.1+, for even better performance, you can enable the option
-`zend-expressive-swoole.swoole-http-server.options.enable_coroutine`. When this
+`zend-expressive-swoole.enable_coroutine`. When this
```

Comment:
As noted in a previous comment, since the PR targets a v2 release, we should create a copy of the v1 docs under a `docs/book/v2/` tree. Update _that_ document, and then also create a `migration.md` document that details the change there, and what steps existing users will need to take to get the original behavior from v1.